worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/adapter/NMSAdapter.java

```diff
@@ -171,6 +175,9 @@ protected static <LevelChunkSection> boolean setSectionAtomic(
      * @since 2.12.0
      */
     protected static void beginChunkPacketSend(String worldName, IntPair pair, StampLockHolder stampedLock) {
+        if (Fawe.isMainThread()) {
```

Comment:
Does that mean we're not sending any packets when on main thread? The callers run
```java
if (lockHolder.chunkLock == null)
    return;
}
```
directly afterwards.

Reply:
I might have been overly eager for the chunk packet methods